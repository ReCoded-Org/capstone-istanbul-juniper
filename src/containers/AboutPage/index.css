.aboutUs {
  background-color: #b4d0c4;
  display: flex;
  flex-direction: column;
}

.aboutUs__upperContent {
  display: flex;
  flex-direction: row;
}

.aboutUs__textCol {
  margin: auto 2%;
  display: flex;
  flex-direction: column;
  width: 45%;
}

.aboutUs__textCol--whyWeDoItParagraph,
.aboutUs__textCol--aboutParagraph {
  text-align: justify;
  margin-bottom: 6%;
}

.aboutUs__textCol--whyWeDoItHeader,
.aboutUs__textCol--aboutHeader {
  font-family: "Salsa", cursive;
}

.aboutUs__illustrationsCol {
  margin: 2%;
  display: flex;
  justify-content: center;
  align-items: center;
  width: 47%;
}

.aboutUs__illustrationsCol--kid0 {
  height: 30%;
  width: auto;
  margin: 0 13px;
}

.aboutUs__illustrationsCol--kid1 {
  height: 30%;
  width: auto;
  margin: 0 13px;
}

@media (max-width: 426px) {
  .aboutUs__upperContent {
    display: flex;
    flex-direction: column;
  }

  .aboutUs__textCol {
    margin-left: 3%;
    margin-right: 3%;
    flex-direction: column;
    width: 94%;
  }

  .aboutUs__illustrationsCol {
    margin-left: 3%;
    margin-right: 3%;
    width: 94%;
  }

  .aboutUs__illustrationsCol--kid1 {
    height: 150px;
    width: auto;
  }

  .aboutUs__illustrationsCol--kid0 {
    height: 150px;
    width: auto;
  }
}
