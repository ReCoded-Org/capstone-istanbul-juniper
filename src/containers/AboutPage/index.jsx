import React from "react";
import { Row, Col } from "antd";
import kid0Img from "./../../images/aboutPage/kid-1.png";
import kid1Img from "./../../images/aboutPage/kid-2.png";
import { useTranslation } from "react-i18next";
import "./index.css";

const AboutPage = () => {
  const [t] = useTranslation();

  return (
    <div className="aboutUs">
      <Row className="aboutUs__upperContent">
        <Col className="aboutUs__textCol">
          <Row className="aboutUs__textCol--aboutText">
            <h2 className="aboutUs__textCol--aboutHeader">
              {t("about.aboutUsHeader")}
            </h2>
            <p className="aboutUs__textCol--aboutParagraph">
              {t("about.aboutUsText")}
            </p>
          </Row>
          <Row className="aboutUs__textCol--whyWeDoIt">
            <h2 className="aboutUs__textCol--whyWeDoItHeader">
              {t("about.whyWeDoItHeader")}
            </h2>
            <p className="aboutUs__textCol--whyWeDoItParagraph">
              {t("about.whyWeDoItText")}
            </p>
          </Row>
        </Col>
        <Col className="aboutUs__illustrationsCol">
          <img
            className="aboutUs__illustrationsCol--kid0"
            src={kid0Img}
            alt="kid-illustration"
          />
          <img
            className="aboutUs__illustrationsCol--kid1"
            src={kid1Img}
            alt="kid-illustration"
          />
        </Col>
      </Row>
      <Row className="team">{/* team members cards are rendered here */}</Row>
    </div>
  );
};

export default AboutPage;
