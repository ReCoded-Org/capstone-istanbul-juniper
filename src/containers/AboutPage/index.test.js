import React from "react";
import AboutPage from "./index.jsx";
import renderer from "react-test-renderer";

describe("AboutPage Component", () => {
  it("matches the tree", () => {
    const tree = renderer.create(<AboutPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
