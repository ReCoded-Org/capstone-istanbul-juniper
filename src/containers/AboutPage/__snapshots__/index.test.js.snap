// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Component matches the tree 1`] = `
<div
  className="aboutUs"
>
  <div
    className="ant-row aboutUs__upperContent"
    style={Object {}}
  >
    <div
      className="ant-col aboutUs__textCol"
      style={Object {}}
    >
      <div
        className="ant-row aboutUs__textCol--aboutText"
        style={Object {}}
      >
        <h2
          className="aboutUs__textCol--aboutHeader"
        >
          about.aboutUsHeader
        </h2>
        <p
          className="aboutUs__textCol--aboutParagraph"
        >
          about.aboutUsText
        </p>
      </div>
      <div
        className="ant-row aboutUs__textCol--whyWeDoIt"
        style={Object {}}
      >
        <h2
          className="aboutUs__textCol--whyWeDoItHeader"
        >
          about.whyWeDoItHeader
        </h2>
        <p
          className="aboutUs__textCol--whyWeDoItParagraph"
        >
          about.whyWeDoItText
        </p>
      </div>
    </div>
    <div
      className="ant-col aboutUs__illustrationsCol"
      style={Object {}}
    >
      <img
        alt="kid-illustration"
        className="aboutUs__illustrationsCol--kid0"
        src="kid-1.png"
      />
      <img
        alt="kid-illustration"
        className="aboutUs__illustrationsCol--kid1"
        src="kid-2.png"
      />
    </div>
  </div>
  <div
    className="ant-row team"
    style={Object {}}
  >
    <div
      className="ourTeam"
    >
      <div
        className="ant-row ourTeam__Header"
        style={Object {}}
      >
        Our Team
      </div>
      <div
        className="ant-row ourTeam__teamCards"
        style={Object {}}
      />
    </div>
  </div>
</div>
`;
